akka-docs/src/main/paradox/typed/persistence-snapshot.md

```diff
@@ -0,0 +1,64 @@
+# Persistence - snapshotting
+
+## Snapshots
+
+As you model your domain using @ref:[persistent actors](persistence.md), you may notice that some actors may be
+prone to accumulating extremely long event logs and experiencing long recovery times. Sometimes, the right approach
+may be to split out into a set of shorter lived actors. However, when this is not an option, you can use snapshots
+to reduce recovery times drastically.
+
+Persistent actors can save snapshots of internal state every N events or when a given predicated of the state
+is fulfilled.
```

Comment:
This is one of my concerns, that it is only by number of events versus snapshot/batch size or time interval options. Would be nice to have a few options available, rather than number of events. Batch size and snapshot size allows are more easily calculable sizing in parallel with provisioning, particularly by data set.

Reply:
Which I am noting here but it's not relevant to this change of course. I have it noted elsewhere.